core/src/main/java/io/github/tcdl/MsbContext.java

```diff
@@ -15,7 +15,10 @@
 import java.time.Clock;
 
 /**
- * {@link  MsbContext} specifies the context of the MSB message processing. Creates singleton beans required for MSB.
+ * Specifies the context of the MSB message processing.
+ * Method {@link MsbContextBuilder#withShutdownHook} is used for specifying shutdown hook during JVM exit.
```

Comment:
At the method more detail descriptions,at the Class level briefly descriptions and link to javadoc for methods.